ntbs-integration-tests/NotificationPages/CreatePageTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using ntbs_integration_tests.Helpers;
+using ntbs_integration_tests.TestServices;
+using ntbs_service;
+using ntbs_service.Helpers;
+using Xunit;
+
+namespace ntbs_integration_tests.NotificationPages
+{
+    public class CreatePageTests : TestRunnerNotificationBase
+    {
+
+        public CreatePageTests(NtbsWebApplicationFactory<Startup> factory) : base(factory)
+        {
+        }
+
+        [Fact]
+        public async Task GetCreate_ReturnsRedirect()
+        {
+            // Arrange
+            using (var client = Factory.WithUserAuth(TestUser.NhsUserForAbingdonAndPermitted)
+                .CreateClientWithoutRedirects())
+            {
+                // Act
+                var response = await client.GetAsync(RouteHelper.GetCreateNotificationPath());
+
+                // Assert
+                Assert.Equal(HttpStatusCode.Redirect, response.StatusCode);
+            }
+        }
+
+        [Fact]
+        public async Task GetCreate_ReturnsNotification_WithDefaultDrugResistanceProfile()
+        {
+            // Arrange
+            using (var client = Factory.WithUserAuth(TestUser.NhsUserForAbingdonAndPermitted)
+                .CreateClientWithoutRedirects())
+            {
+                // Act
+                var createResponse = await client.GetAsync(RouteHelper.GetCreateNotificationPath());
+
+                // Assert
+                var editPage = await Client.GetAsync(createResponse.Headers.Location);
+                var editDocument = await GetDocumentAsync(editPage);
+
+                editDocument.QuerySelector("#patient-details-edit-link");
+                var drugResistanceValue = editDocument
+                    .QuerySelectorAll("div.notification-banner-body div.bold-label")
+                    .FirstOrDefault(e => e.InnerHtml == " Drug resistance profile ")
```

Comment:
Based on our conversation earlier, it sounds like we are going to move to an ID-based approach to reading content in these types of tests. Should I add an ID to just this field in the notification banner (maybe making a helper class like `HtmlIdHelper` or something to store the names of these IDs) or continue with this slightly more fragile approach whilst Leon continues to work on the UI tests?

Reply:
I think you should probably leave this for now, otherwise there are likely to be merge conflicts. You might want to add a TODO against Leon's ticket though.